contrib/test.sh

```diff
@@ -21,20 +27,43 @@ then
 fi
 
 
-# Use toolchain if explicitly specified
-if [ -n "$TOOLCHAIN" ]
-then
-    alias cargo="cargo +$TOOLCHAIN"
-fi
+echo "********* Testing std *************"
+# Test without any features first.  std is required for tests
```

Comment:
If we're just gonna require `std` for tests, I'm a bit confused why a bunch of `#[cfg(test)]` blocks have std/no-std feature checks. Is it just that we intend to support `no_std` tests but don't yet and its WIP to be done in a followup?

Reply:
Comment was wrong, and we do indeed test with `no-std` below.  Fixed.